doc/sphinx/source/frameworks/validation-testing.rst

```diff
@@ -4,97 +4,86 @@
 Validation and testing
 ######################
 
-* **Validating** -- To provide optimizations with nGraph, we first 
-  confirm that a given workload is :term:`validated` as being functional; 
-  that is, we can successfully load its serialized graph as an nGraph 
-  :term:`function graph`. Following here is a list of 14 workloads 
-  we've tested with success.
+We validated perfomance for the following TensorFlow and MXNet workloads:
```

Comment:
Performance is typo.  Also, since this is the first instance of TF and MXNet on this page, probably need the asterisk